DLC/GetIsLoadingScreenActive.md

```diff
@@ -1,5 +1,6 @@
 ---
 ns: DLC
+apiset: client
```

Comment:
```suggestion
```

We don't need to use `apiset` here, all of these natives are assumed to be client.